foundation_cms/legacy_apps/wagtailpages/views.py

```diff
@@ -18,12 +20,24 @@ def custom404_view(request, exception):
     belongs to which site, as "a site" is not tied to "a django
     app" in the wagtail way of things.
     """
-    if Site.find_for_request(request).hostname == "www.mozillafestival.org":
+
```

Comment:
Does pretty much the same thing as "FoundationNavigationPageMixin". The reason we need to add it here is because 404's technically are not pages in the wagtail sense so they are not able to utilize the mixin.